ci/cudf_pandas_scripts/run_tests.sh

```diff
@@ -50,3 +55,10 @@ else
 fi
 
 python -m pytest -p cudf.pandas ./python/cudf/cudf_pandas_tests/
+
```

Comment:
```suggestion
```

As @ajschmidt8 suggests, running with coverage enabled will _also_ run the test suite!

Reply:
My apologies, it was an oversight to not have deleted this.